core-bundle/src/Command/ResizeImagesCommand.php

```diff
@@ -131,7 +131,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $concurrent = (float) $input->getOption('concurrent');
 
         if (false !== $input->getOption('throttle')) {
-            @trigger_error('Using the throttle option is deprecated and will no longer work in Contao 5.0. Use the concurrent option instead.', E_USER_DEPRECATED);
+            trigger_deprecation('contao/core-bundle', '4.10', 'Using the throttle option is deprecated and will no longer work in Contao 5.0. Use the concurrent option instead.', E_USER_DEPRECATED);
```

Comment:
```suggestion
            trigger_deprecation('contao/core-bundle', '4.9.5', 'Using the throttle option is deprecated and will no longer work in Contao 5.0. Use the concurrent option instead.', E_USER_DEPRECATED);
```

Reply:
So far, we have only provided the major and minor version. Would `'4.9'` suffice as well or is it important to also specify the bugfix version?

Reply:
`4.9` is fine for me